### PR TITLE
Update supportability information for container runtimes

### PR DESCRIPTION
**Pull request by inputoutputcode**

### CODE REVIEW

File: Deployment/Container-Support-Guidance.md [hunk 1]
```diff
@@ -264,8 +267,9 @@ Documentation:
 
 For all further questions please reach out to your account team or [create a Microsoft support case](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request).
 
-1. Is Mirantis Container Runtime the only container runtime supported by Service Fabric?
+1. Which container runtimes are supported by Service Fabric?
 
-   The Service Fabric product group is currently testing Moby. This will be supported soon. Please stay tuned for the official announcement.
-   Moby is the open-source version of the container runtime, former DockerCE. It comes with community support only and is suitable for non-production environments.
+   Microsoft validated Service Fabric 9.0 CU1 or later with Mirantis Container Runtime v20.10.13 and Moby v20.10.18 on Windows Server 2019/2022. Please make yourself familiar with the support options of these container runtimes.
```
Comment (anaahatsda) [inline Deployment/Container-Support-Guidance.md:272]: nit : support options FOR these customers

File: Deployment/Container-Support-Guidance.md [hunk 2]
```diff
@@ -264,8 +267,9 @@ Documentation:
...
+   - Moby is an open-source container runtime, former DockerCE.
```
Comment (anaahatsda) [inline Deployment/Container-Support-Guidance.md:273]: Moby is an open framework to assemble specialized container systems

**Review by anaahatsda — CHANGES_REQUESTED** [review]
Provided comments

**Review by sedwardstx — APPROVED** [review]

**Review by sedwardstx — COMMENTED** [review]
changes look good.  Since we are renaming the original .md files, we may be breaking external links.  Do we want to add back the original file with a fwd link to the current doc?

**Pull request merged**